applications/crossbar/src/modules/cb_recordings.erl

```diff
@@ -189,11 +260,11 @@ get_disposition(MediaName, Context) ->
 normalize_view_results(JObj, Acc) ->
     [kz_json:public_fields(kz_json:get_value(<<"doc">>, JObj))|Acc].
 
--spec get_view_and_filter(cb_context:context()) -> {ne_binary(), api_binaries(), api_binaries()}.
+-spec get_view_and_filter(cb_context:context()) -> {ne_binary(), api_binaries(), api_objects()}.
```

Comment:
Should be [binary()] as the third tuple element in the tuple. But since it doesn't vary at all, maybe we just omit returning it and use the `[<<>>]` literal as the `PostFilter`